docs/modules/sql/pages/partition-pruning.adoc

```diff
@@ -0,0 +1,63 @@
+= Partition pruning of SQL queries
+:description: The partition pruning optimization technique reduces latency for faster IMap SQL queries.
+:page-beta: false
+
+Partition pruning is an IMap-specific query optimization technique that automatically prunes members that don't contain
+the required data on their partitions, and optimizes the count of Jet scan processors on required members. It works:
+
+* by default, for primitive `__key` types (default attribute strategy).
+* for all key types if you configure the `AttributePartitioningStrategy` in `MapConfig`.
+
```

Comment:
Suggest amalgamating the notes, as follows:

```
[NOTE] 
====
. You can only use partition pruning if the partition strategy attribute is enabled.
. The query.....
====
```

Reply:
Is there/will there be documentation for the AttributePartitionStrategy related config? If so, a link would be good

Reply:
I made a quick search, and doc for `AttributePartitionStrategy` doesn't exist yet.